flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/over/BufferDataOverWindowOperatorTest.java

```diff
@@ -205,9 +206,19 @@ public StreamConfig getOperatorConfig() {
 
 			@Override
 			public StreamTask<?, ?> getContainingTask() {
-				StreamTask task = mock(StreamTask.class);
+				class MockableStreamTask extends StreamTask<Object, StreamOperator<Object>> implements EnvironmentSupport {
+					MockableStreamTask() throws Exception {
+						super(new MockEnvironmentBuilder().build());
+					}
+
+					@Override
+					protected void init() {
+					}
+				}
+
+				StreamTask<?, ?> task = mock(MockableStreamTask.class);
```

Comment:
TBH i did not get the point why this class change is related to the commit motivation. 
If it is necessary, can we use `MockStreamTaskBuilder` to build `MockStreamTask` instead, to avoid construct `MockableStreamTask`. Besides that, it is not suggested to use `mock` in unit tests.

Reply:
After making `final` some methods of `StreamTask` this test broke because mockito wasn't able now to override them.
Eventually, I removed mockito from here.